src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -50,18 +50,7 @@ protected override void Awake()
             // messages even if it loses focus
             Application.runInBackground = true;
 
-            if (connectionManager != null)
-            {
-                DebugLog("Setting up connection manager");
-
-                // Start listening to incoming connections as well.
-                connectionManager.StartListening(port);
-            }
-            else
-            {
-                Debug.LogError("Connection manager not specified for Observer.");
-            }
-
+            StartListening(port);

```

Comment:
IIRC this listening socket is unused. Can we remove it since we are touching this code?

Reply:
i would prefer to keep this for now, i'm not sure the impact removing this will have on all video camera/editor connection items.

Reply:
@matthejo  any thoughts on whether or not this socket is used?